.github/workflows/pytest.yml

```diff
@@ -42,23 +42,22 @@ jobs:
           --health-retries 5
 
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Install system dependencies
-        run: apt-get install -y libpq-dev
+       - name: Install and configure poetry
+        run: |
+          pipx install poetry==1.5.0
+          poetry -m config virtualenvs.create true
```

Comment:
`virtualenvs.create = true` seems redundant as it's already the default config value. Or is this here for some other reason?

Reply:
@NyanKiyoshi Thank you for the review. The whole point of the command was to change the poetry config value. But as you rightly suggested, since the default config is already `true`, there is no point of using the command in workflow actions.